CHANGELOG.unreleased.md

```diff
@@ -25,3 +25,4 @@ Section headings should be at level 3 (e.g. `### Added`).
 
 - Fixed ValueError on Windows when running a W&B script from a different drive (@jacobromero in https://github.com/wandb/wandb/pull/9678)
 - Fix base_url setting was not provided to wandb.login (@jacobromero in https://github.com/wandb/wandb/pull/9703)
+- Fixed `util.get_module(..., lazy=True)` causing AttributeError when accessing a submodule from its parent (@jacobromero in https://github.com/wandb/wandb/pull/9598)
```

Comment:
This is a private function, so it shouldn't be in the changelog. The changelog should describe the user-facing issue that got fixed.